modules/builds-strategy-docker-squash-layers.adoc

```diff
@@ -17,4 +17,4 @@ strategy:
   dockerStrategy:
     imageOptimizationPolicy: SkipLayers <1>
 ----
-<1> Layers are always squashed in {product-title} {product-version}.
+<1> Builds will keep their layers by default. 
```

Comment:
Personally I would just get rid of this footnote, since we say the same thing above:

> Docker builds normally create a layer representing each instruction in a
`Dockerfile`. Setting the `imageOptimizationPolicy` to `SkipLayers` will merge
all instructions into a single layer on top of the base image.

Reply:
Easy enough!